src/palace/manager/api/sip/__init__.py

```diff
@@ -192,6 +195,19 @@ class SIP2LibrarySettings(BasicAuthProviderLibrarySettings):
             description="A specific identifier for the library or branch, if used in patron authentication",
         ),
     )
+    # Used with SIP2, when it is available in the patron information response.
+    patron_location_restriction: str | None = FormField(
+        None,
+        form=ConfigurationFormItem(
+            label="Patron Location Restriction",
+            description=(
+                "A code for the library or branch which, when specified, "
+                "must exactly match the permanent location for the patron."
+                "<br>If an ILS does not include a location for patrons, specifying"
```

Comment:
I think you need a space before the closing quote if I'm not mistaken.

Reply:
Good catch! Thanks! We do, indeed.

Reply:
Fixed!